messaging/writer_test.go

```diff
@@ -33,13 +33,26 @@ func TestMissyWriter_WriteSuccess(t *testing.T) {
 
 	writer := missyWriter{brokerWriter: brokerWriterMock}
 
+	exec := false
+
+	monkey.PatchInstanceMethod(reflect.TypeOf(brokerWriterMock), "WriteMessages", func(_ *MockBrokerWriter, ctx context.Context, messages ...Message) error {
```

Comment:
Please do the test like this:
1. create time := time.Now().UTC()
2. patch time.Time and return above patch
3. in PatchInstanceMethod ("WriteMessages") Check if all fields are set properly (key, value and time)